### DISCUSSION TITLE
How can I save worksheet on other workbook file?

### DISCUSSION THREAD

**Discussion by yuke0929** [exceljs/exceljs#2500] [Q&A]
I edited a worksheet and I want to save this worksheet on a new excel file. But I can't find any method like workbook.setWorksheet(worksheet). Is there any idea about moving one worksheet from an old excel file to a new excel file?

**Comment by y0m0** [comment]
There is a read and a write example in the [Docs](https://github.com/guyonroche/exceljs#writing-xlsx)
Anyway you can do something like this:
```
var workbook = new Excel.Workbook();
workbook.xlsx.readFile(filename)
    .then(() => {
        // use workbook
    })
    .then(() => {
        workbook.xlsx.writeFile(newFilename)
            .then(() => console.log('file created'))
    })
```
